data/json/items/tool_armor.json

```diff
@@ -2077,6 +2087,44 @@
     "coverage": 10,
     "material_thickness": 2
   },
+  {
+    "type": "TOOL_ARMOR",
+    "id": "guitar_amplifier",
+    "name": "amplifier electric guitar",
```

Comment:
```suggestion
    "name": "electric guitar amplifier",
```

Reply:
Yeah, my naming of that item was too ambiguous. In order to not get the item confused with an actual guitar amplifier, I renamed it to "amplified electric guitar".